meson.build

```diff
@@ -367,13 +367,24 @@ if get_option('enable-app')
   endif
 endif
 
-if get_option('enable-nnstreamer-tensor-filter')
-  if get_option('platform') == 'android'
-    warning('android nnstreamer-filter is not yet supported, building nnstreamer-filter skipped')
-  else
-    nnstreamer_dep = dependency('nnstreamer', required: true)
-    subdir('nnstreamer/tensor_filter')
-  endif
+if get_option('enable-nnstreamer-tensor-filter').enabled()
+	if get_option('platform') == 'android'
+		warning('android nnstreamer-filter is not yet supported, building nnstreamer-filter skipped')
+	else
+    nnstreamer_filter_dep = dependency('nnstreamer')
+		subdir('nnstreamer/tensor_filter')
+		extra_defines += '-DSUPPORT_NNSREAMER_PLUGIN_FILTER=1'
+	endif
+endif
+
+if get_option('enable-nnstreamer-tensor-trainer').enabled()
```

Comment:
I do not see any reason why these two options are separated. I'm thinking if nnstreamer is enabled.. then those inference and training options should be enabled. Could you check?

Reply:
Here is @myungjoo 's comment:
Since tensor_trainer is unstable,
It is better to separate the build options so that tensor_trainer only can be removed when nntrainer is released.
How about you?

Reply:
As agreed with each other,
When the tensor_trainer sub-plugin is stabilized, we will merge it with the tensor_filter sub-plugin package.